eclipsePlugin/about.properties

```diff
@@ -1,4 +1,6 @@
 blurb=SpotBugs \n\
 \n\
-(c) SpotBugs Project. Licensed under the LGPL.\n\
-https://spotbugs.github.io\n\
+Copyright (c) 2017 SpotBugs Project. All rights reserved.\n\
+Licensed under the LGPL.\n\
+Copyright 2003-2008 by the University of Maryland. \n\
+Visit https://spotbugs.github.io\n\
```

Comment:
The URI should end with a trailing slash: `https://spotbugs.github.io/` (this is the canonical form)